src/main/java/org/sunbird/user/registration/util/UserRegistrationStatus.java

```diff
@@ -8,15 +8,22 @@
  */
 public enum UserRegistrationStatus {
 
-	CREATED(1), WF_INITIATED(2), WF_APPROVED(3), WF_DENIED(4), FAILED(5);
+	CREATED(1, "Created"), WF_INITIATED(2, "Initiated"), WF_APPROVED(3, "Approved"), WF_DENIED(4, "Denied"), FAILED(5,
+			"Failed");
 
 	private int status = 0;
+	private String name;
 
-	UserRegistrationStatus(int status) {
+	UserRegistrationStatus(int status, String name) {
 		this.status = status;
+		this.name = name;
 	}
 
 	public int getStatus() {
 		return status;
 	}
+
+	public String getName() {
+		return name;
+	}
```

Comment:
Enum has the Name attribute by default. Current implementation should be :
	public String getName() {
		return this.name();
	}